restclient/RestClient.hx

```diff
@@ -119,6 +128,17 @@ class RestClient
             }
         }
         
+        // Add header for non-standard HTTP verbs
+	#if sys
+	var useOverride = RestClient.useHttpOverride;
+	#else
+	var useOverride = true;
```

Comment:
Minor: 4-space indentation instead of tabs

Reply:
Fixed in 01c3594